### PR TITLE
Setup and requirements

### PR DESCRIPTION
**Pull request by mkatsimpris**
Created the requirements.txt and setup.py according to the best practices in http://discourse.myhdl.org/t/best-practices-for-cores-development-gsoc-2016-projects/30

@cfelton, @jck, @josyb, @dargor0, @nkkav

### CODE REVIEW

File: jpegenc/setup.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python
+
+from setuptools import setup
+
+setup(
+    name='JPEG Encoder',
+    version='0.1dev',
+    author='Merkourios Katsimpris',
+    author_email='merkourioskatsimpris@gmail.com',
+    url='https://github.com/mkatsimpris/test_jpeg',
```
Comment (cfelton) [inline jpegenc/setup.py]: This looks good, we will want to merge this into `cfelton/test_jpeg` so we can collaborate per the emails.  The url should point to the original repo for now.

```
url=https://github.com/cfelton/test_jpeg
```

I am not sure how the author typically work, if it should list the three of us or single author is fine.

In addition to merging to your `master` these changes can pushed to original repo.

**Comment by cfelton** [comment]
@mkatsimpris you might want to merge your `master` to the last then your PR will only show the modifications you made. 

I believe what happened is your `master` branch lags and you updated (sync'd) the branch you are developing on.  Then the PR shows the changes made to the remote repo and your changes.  I image the requirements.txt changes did require changes to 24 files :)

File: jpegenc/setup.py (hunk 1 above)
Comment (mkatsimpris) [inline jpegenc/setup.py]: ok will change the url and make a pull reguest to your repo right?

@cfelton

**Comment by mkatsimpris** [comment]
I have merged my master branch with your master branch in your repo. I dont know why it still show 24 file changes

**Comment by cfelton** [comment]
@mkatsimpris can you merge this and make a PR to the original?  Or once you merge the PR I can pull from your master and merge.

**Comment by mkatsimpris** [comment]
i will do it

**Comment by cfelton** [comment]
@mkatsimpris I didn't notice at first but these files need to be moved up one directory (requirements.txt and setup.py)

**Comment by mkatsimpris** [comment]
So the setup.py and the requirements.txt wil be in the test_jpeg folder instead of jpegenc?

**Pull request merged**